Readme.md

```diff
@@ -22,17 +22,25 @@ To build it (most of these instructions come from [here](https://gcc.gnu.org/onl
 
 ```bash
 $ git clone https://github.com/antoyo/gcc
-$ cd gcc
 $ sudo apt install flex libmpfr-dev libgmp-dev libmpc3 libmpc-dev
-$ ./configure \
-   --enable-host-shared \
-   --enable-languages=jit \
-   --disable-bootstrap \
-   --enable-checking=release \
-   --prefix=$(pwd)/install \
-   --disable-multilib
+$ mkdir gcc-build gcc-install
+$ cd gcc-build
+$ ../gcc/configure \
+    --enable-host-shared \
+    --enable-languages=jit,c++ \
```

Comment:
Please the `c++` argument and mention below that this is required only to run gccjit's tests.